Update 16.2021.1 - Disputes (#34)
Epride/wca-documents: documents/motions/16.2021.1 - Disputes.md

# SUBMISSION OF PROPOSED MOTION

**Motion number:** 16.2024.1  
**Subject:** Disputes  
**Intent:** Definition of the Processes in case of Disputes  
**Submitted by:** WCA Board  
**Date:** TBC, 2024  

# Motion

1. Acts and decisions of WCA Staff may be subject to disputes by affected individuals or organizations.
   1. Disputes regarding the application of WCA Regulations must be directed to and handled by WCA Regulations Committee, and are bound by the provisions of the WCA Regulations.

>    2. Disputes regarding the conduct of WCA Staff where it is alleged the Spirit, Mission, Motions, Policies, Codes, Regulations and the applicable governing laws are violated must be directed to and handled by WCA Integrity Committee, and are bound by the Spirit, Mission, Motions, Policies, Codes, Regulations of the WCA, and the applicable governing laws.

Reviewer: should it be "are" or "were"? (I think it should be were)

Reviewer: same applies for 1.3

   3. Disputes regarding the conduct of WCA Community members where it is alleged the Spirit, Mission, Motions, Policies, Codes, Regulations and the applicable governing laws are violated must be directed to and handled by WCA Integrity Committee, and are bound by the Spirit, Mission, Motions, Policies, Codes, Regulations of the WCA, and the applicable governing laws. 
   4. All other Disputes should be directed to and handled by WCA Integrity Committee, and are bound by the Spirit, Mission, Motions, Policies, Codes, Regulations of the WCA, and the applicable governing laws.

> 2. All disputes may be subject to appeal by affected individuals or organizations.

Reviewer: Instead of saying "all disputes may be subject to appeal" wouldn't it be better to say "The decision made by the Committee handling the dispute may be subject to appeal ... " - you're not appealing the dispute, you're appealing the decision made about it

   2. Disputes regarding the conduct of WCA Staff where it is alleged the Spirit, Mission, Motions, Policies, Codes, Regulations and the applicable governing laws were violated must be directed to and handled by WCA Integrity Committee, and are bound by the Spirit, Mission, Motions, Policies, Codes, Regulations of the WCA, and the applicable governing laws.
   3. Disputes regarding the conduct of WCA Community members where it is alleged the Spirit, Mission, Motions, Policies, Codes, Regulations and the applicable governing laws were violated must be directed to and handled by WCA Integrity Committee, and are bound by the Spirit, Mission, Motions, Policies, Codes, Regulations of the WCA, and the applicable governing laws. 
   4. All other Disputes should be directed to and handled by WCA Integrity Committee, and are bound by the Spirit, Mission, Motions, Policies, Codes, Regulations of the WCA, and the applicable governing laws.
2. The decision made by the Committee handling the dispute may be subject to appeal by affected individuals or organizations.

>    1. Appeals on disputes shall be handled by the WCA Appeals Committee.

Reviewer 2: Any reason to use "shall" instead of "must" (which is usually the preferred option for WCA Documents)?

Considering RFC 2119, I assume both have the same meaning, but it could be ambiguous for anyone used to "may", "should" and "must".

>    1. Appeals on disputes shall be handled by the WCA Appeals Committee.

Reviewer 2: ```suggestion
   1. Appeals on disputes must be handled by the WCA Appeals Committee.
```

Any reason to use "shall" instead of "must" (which is usually the preferred option for WCA Documents)?

Considering RFC 2119, I assume both have the same meaning, but it could be ambiguous for anyone used to "may", "should" and "must".

3. Disputes must be filed within sixty days after the disputed act or decision.
4. Disputes must be concluded in a timely manner
   1. Once the Committee handling the dispute has made their final decision the result of the dispute is communicated to the all interested parties. The dispute is considered concluded once this communication is complete.
5. Appeals to decisions on disputes must be filed within sixty days after the dispute is concluded.
6. Appeals must be concluded in a timely manner

>    1. An appeal is considered concluded when the outcome has been communicated to all interested parties as per 5.4

Reviewer 3: Now 6.4. Also missing a full stop at the end (as are 2.1, 4.0 and 6.0)

   2. The WCA Appeals Committee initiates the appeal by consulting the public evidence and, if applicable, the report of the Original Committee.
      1. The WCA Appeals Committee should contact the Original Committee and ask them to disclose the evidence used to make their final decision.

>       2. The WCA Appeals Committee investigates the correct handling of the case by the original Committee.

Reviewer 2: Lowercase "original".

>       2. The WCA Appeals Committee investigates the correct handling of the case by the original Committee.

Reviewer 2: ```suggestion
      2. The WCA Appeals Committee investigates the correct handling of the case by the Original Committee.
```

      3. The WCA Appeals Committee may interview all the witnesses and evaluate all the evidence considered by the Original Committee, as well as find new witnesses and new evidence.
      4. The WCA Appeals Committee may refuse to initiate an appeal if they deem there is no valid ground for appeal.
   3. The WCA Appeals Committee may consider the appeal valid only if one or more of the following conditions are met:

>       1. The WCA Appeals Committee determines that the original Committee did not conduct a proper decision making process in making their final decision. This includes but is not limited to negligence and conflict of interest.

Reviewer 2: Lowercase "original" too.

>       1. The WCA Appeals Committee determines that the original Committee did not conduct a proper decision making process in making their final decision. This includes but is not limited to negligence and conflict of interest.

Reviewer 2: ```suggestion
      1. The WCA Appeals Committee determines that the Original Committee did not conduct a proper decision making process in making their final decision. This includes but is not limited to negligence and conflict of interest.
```

>       2. The decision taken by the original Committee is not in accordance with the WCA Regulations, Motions, Policies or Bylaws.

Reviewer 2: ```suggestion
      2. The decision taken by the Original Committee is not in accordance with the WCA Regulations, Motions, Policies or Bylaws.
```

>       3. The WCA Appeals Committtee uncovers new evidence that would have caused the original Commmittee to reach a different conclusion.

Reviewer 2: "Commi**ttt**ee" and "original".

>       3. The WCA Appeals Committtee uncovers new evidence that would have caused the original Commmittee to reach a different conclusion.

Reviewer 2: ```suggestion
      3. The WCA Appeals Committee uncovers new evidence that would have caused the Original Committee to reach a different conclusion.
```

   4. Once the WCA Appeals Committee has made their final decision the result of the appeal is communicated to the appealer, the Original Committee and all other interested parties.

>       1. A report of the appeal is sent to the Original Committee and the Board. 

Reviewer 4: Considering there is reference to the Board of Appeal, I think it clearer for it to say WCA Board.
